depends/packages/qt.mk

```diff
@@ -133,9 +133,6 @@ $(package)_config_opts_darwin += -no-feature-corewlan
 $(package)_config_opts_darwin += -no-freetype
 $(package)_config_opts_darwin += QMAKE_MACOSX_DEPLOYMENT_TARGET=$(OSX_MIN_VERSION)
 
-# Optimizing using > -O1 causes non-determinism when building across arches.
```

Comment:
28778 - should we test it?

Reply:
CI on this branch gave
```
7e50766d89e933bf281fca317d0d0570595288535d1c0a9c7ab78d9ab62850e8  arm64-apple-darwin/dashcore-21.1.1-1146-gc75a0d4c575e-arm64-apple-darwin-debug.tar.gz
b268e492b334518c2ae7bc57a06db7c36e56bdaaae9806f1129f2fe33edb9f12  arm64-apple-darwin/dashcore-21.1.1-1146-gc75a0d4c575e-arm64-apple-darwin-unsigned.dmg
21c44d30c5664728f1a4cab8861c3a65aced61e5b838b33cfa5b1e8bff841570  arm64-apple-darwin/dashcore-21.1.1-1146-gc75a0d4c575e-arm64-apple-darwin-unsigned.tar.gz
0610721bc281e13581564fb1134c27a66d9e6a7ed2986f6c134fc2e5cd7fe495  arm64-apple-darwin/dashcore-21.1.1-1146-gc75a0d4c575e-arm64-apple-darwin.tar.gz
9f2d68a3584ab572c963570c3dffbdfe4ac7ce7f2ab477ef6330bc98f7ea5e24  dist-archive/dashcore-21.1.1-1146-gc75a0d4c575e.tar.gz
```

will provide hashes from my repo once complete

Reply:
We should; I think let's merge it in and then we'll build guix here and on my repo, and compare. Revert if needed